contrib/mesos/pkg/scheduler/plugin_test.go

```diff
@@ -362,28 +371,30 @@ func newTaskStatusForTask(task *mesos.TaskInfo, state mesos.TaskState) *mesos.Ta
 	}
 }
 
-// Test to create the scheduler plugin with an empty plugin config
-func TestPlugin_New(t *testing.T) {
-	assert := assert.New(t)
+type LaunchedTask struct {
```

Comment:
In this package, no (good catch). If we refactor in a separate `test` package, then I'd vote for public visibility.